### PR TITLE
Added description of Seccomp configuration options.

### PR DESCRIPTION
**Pull request by abudnik**
## Description
https://jira.mesosphere.com/browse/DCOS_OSS-3870

Add docs about new installer config variables.

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md [hunk 1]
```diff
@@ -499,6 +501,20 @@ The location of the Mesos work directory on master nodes. This defines the `work
...
+*  `mesos_seccomp_enabled: 'true'` Enables Seccomp isolator on Mesos agents. Seccomp isolator is used to set up a Seccomp profile for UCR containers.
+*  `mesos_seccomp_enabled: 'false'` Seccomp are not available for use in the cluster. This is the default value.
+
+For more information, see the [Seccomp documentation](http://mesos.apache.org/documentation/latest/isolators/linux-seccomp/).
+
+**Note**: DC/OS provides a default Seccomp profile, which can be enabled for UCR containers via the [`mesos_seccomp_profile_name`](#mesos-seccomp-profile-name) option.
+
+### mesos_seccomp_profile_name
+
+Specifies the name of the default Seccomp profile which is applied for UCR containers. If you set this configuration option to `default.json`, Mesos agents will use built-in Seccomp profile. This profile is a slightly modified version of a Docker default profile. It can be found in `/opt/mesosphere/etc/dcos/mesos/seccomp`. We would strongly recommend to stick with the built-in default Seccomp profile.
```
Comment (Gilbert88) [inline pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md]: s/UCR containers/cluster wide UCR containers/g?

File: pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md [hunk 2]
```diff
@@ -499,6 +501,22 @@ The location of the Mesos work directory on master nodes. This defines the `work
...
+*  `mesos_seccomp_enabled: 'true'` Enables Seccomp isolator on Mesos agents. Seccomp isolator is used to set up a Seccomp profile for UCR containers.
+*  `mesos_seccomp_enabled: 'false'` Seccomp are not available for use in the cluster. This is the default value.
+
+For more information, see the [Seccomp documentation](http://mesos.apache.org/documentation/latest/isolators/linux-seccomp/).
+
+**Note**: DC/OS provides a default Seccomp profile, which can be enabled for UCR containers via the [`mesos_seccomp_profile_name`](#mesos-seccomp-profile-name) option.
+
+### mesos_seccomp_profile_name
+
+Specifies the name of the default Seccomp profile which is applied cluster-wide for UCR containers. If unset, a Seccomp profile is not applied by default. If you set this configuration option to `default.json`, Mesos agents will use built-in Seccomp profile. This profile is a slightly modified version of a Docker default profile. It can be found in `/opt/mesosphere/etc/dcos/mesos/seccomp`. We would strongly recommend to stick with the built-in default Seccomp profile.
```
Comment (anithamuthu79) [inline pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md]: Does the meaning of unset here refer to disabled? If yes, please use "disabled".

Rephrase: "We would strongly recommend to stick with the built-in default Seccomp profile" by copy-paste the following content:

<p class="message--note"><strong>NOTE: </strong>It is highly recommended to use the built-in default Seccomp profile</p>

Comment (jdef) [reply pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md]: pretty sure that "unset" is the intended word here. i glanced at the dcos configuration and BOTH of the following conditions must exist to apply the default seccomp configuration:

1.  "_enabled" flag set to `true`
2. "_profile_name: flag set to `default.json`

Comment (abudnik) [reply pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md]: > Does the meaning of unset here refer to disabled?

By default, the value of `mesos_seccomp_profile_name` is an empty string `''`. DC/OS interprets an empty string as `default profile is not specified, thereby DC/OS won't apply a default Seccomp profile for UCR containers`. However, any framework can specify a Seccomp profile for a particular container (not implemented yet). So, in theory, Seccomp as a DC/OS feature might be enabled (when `mesos_seccomp_enabled` is `true`), while the default Seccomp profile is not applied by default.

> It is highly recommended to use the built-in default Seccomp profile.

Agree. Updated PR.

**Comment by abudnik** [comment]
> I'd really like to see a summary of the syscalls that excluded by the default profile, and the reason for doing so.

@jdef The requirement was to implement a fully Docker-compatible Seccomp feature. DC/OS default Seccomp profile is a recent Docker Seccomp profile + one extra syscall (`pivot_root`) on top. Hence, the links you have provided are related to DC/OS built-in Seccomp profile. We exclude the same syscalls that default Docker profile does.

**Comment by anithamuthu79** [comment]
LGTM.

**Pull request merged**